frontend/src/utils/attribution-html.ts

```diff
@@ -124,6 +124,21 @@ const licenseElementImg = (licenseElement: LicenseElement): string => {
 const extLink = (href: string, text: string) =>
   h("a", { target: "_blank", rel: "noopener noreferrer", href }, [text])
 
+/**
+ * Generate a Dublin Core conforming XML attribution snippet.
+ *
+ * @param mediaItem - the media item being attributed
+ * @returns the XML markup of the attribution
+ */
+const xmlAttribution = (mediaItem: Record<string, string>) =>
+  `<attribution xmlns:dc="http://purl.org/dc/elements/1.1/">
+	<dc:creator>${mediaItem.creator}</dc:creator>
+	<dc:title>${mediaItem.title}</dc:title>
+	<dc:rights>"${mediaItem.title}" ${mediaItem.byCreator} ${mediaItem.markedLicensed} ${mediaItem.license}. ${mediaItem.viewLegal}</dc:rights>
```

Comment:
Could use `mediaItem.attribution` here for simplicity's sake. I think that catches some potential edge cases that exist? You'd need to add `attribution` to `AttributableMedia` as well (like you did for `frontendMediaType`).

```suggestion
	<dc:rights>${mediaItem.attribution}</dc:rights>
```